core/templates/dev/head/components/html_select_directive.html

```diff
@@ -0,0 +1,14 @@
+<script type="text/ng-template" id="components/html_select">
+  <div class="oppia-html-select" >
+    <button class="btn btn-default dropdown-toggle oppia-html-select-selection" type="button" id="menu1" aria-expanded="false" data-toggle="dropdown">
```

Comment:
Hm, I'm not sure about this. Which other directive did you follow? (I don't think our directives generally have id attributes in their html tags.) What happens if you remove it?